<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transactions_UI</class>
 <widget class="QDialog" name="Transactions_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_transactions">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>450</width>
     <height>450</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>450</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="dateComboBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="amountComboBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Show all</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Show 10</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TRANSACTIONS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
